app/js/components/manageApps/ManageSettings.jsx

```diff
@@ -87,8 +87,9 @@ export default class ManageSettings extends React.Component {
     let {
       AppBaseURLValue,
       AppFolderPathValue,
-      AppStoreURLValue
+      AppStoreURLValue,
```

Comment:
Is the above comma needed?

Reply:
Yes there is always a comma at the end of each object literal